bin/create-webextension

```diff
@@ -1,3 +1,20 @@
 #!/usr/bin/env node
+const chalk = require("chalk");
+const UsageError = require("../errors").UsageError;
+const onlyInstancesOf = require("../errors").onlyInstancesOf;
 
-require("..").main();
\ No newline at end of file
+
+const USAGE_MSG = `Usage: create-webextension project_dir_name`;
+
+if (!process.argv[2]) {
+  console.error(`${chalk.red("Missing project dir name.")}\n`);
+  console.log(USAGE_MSG);
+  process.exit(1);
+}
+
+require("..").main(process.argv[2])
+  .then(console.log)
+  .catch(onlyInstancesOf(UsageError, (error) => {
```

Comment:
@saintsebastian it would be better to use console.error here (so that the logs are going to be printer in the stderr instead of stdout).

also, after this catch we should handle the other errors by chaining another catch:

```js
  .catch(onlyInstancesOf(UsageError, (error) => {
    ...
  })
  .catch((error) => {
    console.error(`${chalk.red(error.message)}: ${error.stack}\n`); // or something similar that prints both the message and the stack
    process.exit(1);
  });
```